### PR TITLE
Fix timeout issue of NESTED-KVM-BASIC-VERIFICATION

### PR DESCRIPTION
**Pull request by lizzha**

### CODE REVIEW

File: XML/TestCases/NestedKvmTests.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
             <param>NestedUserPassword='NESTED_VM_PASSWORD'</param>
             <param>HostFwdPort=60022</param>
         </TestParameters>
+        <Timeout>720</Timeout>
```
Comment (juhlee-microsoft) [inline XML/TestCases/NestedKvmTests.xml]: This test directly calls Bash script, and I can not find timeout value in nested_kvm_basic_test.sh. Instead, there is 'retry_times' variable with value 24. Help me understand this test.

Comment (iamshital) [reply XML/TestCases/NestedKvmTests.xml]: @juhlee-microsoft 
https://github.com/LIS/LISAv2/blob/master/Libraries/Test-Framework.psm1#L485

Comment (iamshital) [reply XML/TestCases/NestedKvmTests.xml]: @lizzha Is this timeout safe for smaller Azure VM sizes also? (1 core / 2 core VMs)

Comment (lizzha) [reply XML/TestCases/NestedKvmTests.xml]: yes, the minimum size supported is Standard_E2_v3, I've verified it's OK

Comment (juhlee-microsoft) [reply XML/TestCases/NestedKvmTests.xml]: acked

**Pull request merged**